src/libraries/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -327,6 +335,31 @@ public Task<Stream> ReadAsStreamAsync(CancellationToken cancellationToken)
         // on all known HttpContent types, waiting for the request content to complete before completing the SendAsync task.
         internal virtual bool AllowDuplex => true;
 
+        public void CopyTo(Stream stream, TransportContext? context, CancellationToken cancellationToken)
+        {
+            CheckDisposed();
+            if (stream == null)
+            {
+                throw new ArgumentNullException(nameof(stream));
+            }
+
+            try
```

Comment:
This is a public API, although improbable, user can call `CopyTo`. Should we add the same `CancellationToken.Register` and `stream.Dispose` here as well? Though it would be duplicate registration for when content is sent through `HttpConnection`.

Reply:
> Should we add the same CancellationToken.Register and stream.Dispose here as well? 

No.  We don't know what the target Stream is and don't own it; Dispose'ing it would mean we were disposing the arbitrary stream handed to us.